src/Sylius/Bundle/UiBundle/Resources/views/Menu/top.html.twig

```diff
@@ -41,6 +43,26 @@
     </a>
 {% endblock %}
 
+{% block links %}
+    {% set color = item.labelAttribute('color') %}
+    {% set icon = item.labelAttribute('icon') %}
+
+    <div class="ui{% if icon %} labeled icon{% endif %} floating dropdown {% if color %}{{ color }} {% endif %}link button">
+        {% if icon %}<i class="{{ icon }} icon"></i>{% endif %}
+        <span class="text">{{ item.label|trans }}</span>
+        <div class="menu">
+            {% for child in item.children %}
+                {% set icon = child.labelAttribute('icon') %}
+
+                <a href="{{ child.uri }}" class="item">
+                    {% if icon %}<i class="icon {{ icon }}"></i> {% endif %}
```

Comment:
{% if icon %}<i class="icon {{ icon }}"></i><cmd-g>{% endif %}

Redundant blank space

Reply:
This space is intentionally missing

Reply:
This space is needed to have space between label and icon I suppose. ❓